trunk/SUAVE/Methods/Weights/Correlations/Common/landing_gear.py

```diff
@@ -29,9 +31,11 @@ def landing_gear(TOW, landing_gear_wt_factor = 0.04):
             
     Properties Used:
         N/A
-    """ 
-    
-    #process
-    weight = landing_gear_wt_factor * TOW
-    
-    return weight
\ No newline at end of file
+    """
+
+    # process
+    weight          = landing_gear_wt_factor * vehicle.mass_properties.max_takeoff
+    output          = Data()
+    output.main     = weight
+    output.nose     = 0
```

Comment:
This is because the method used here combines the weight of the main landing gear and the nose landing gear into one estimate. I wasn't sure what else I could do, we could say the main landing gear is 80% of the weight and the nose landing gear is 20% of the lumped sum (approximately what I found in literature). Let me know what you suggest.

Reply:
This is fine for now, we'll see what the larger group thinks